docs/understanding-airbyte/file-transfer.md

```diff
@@ -0,0 +1,68 @@
+# Airbyte File Sync
+
+Airbyte File Sync is a capability that allows you to move unstructured data, non-text data, and compressed files between sources and destinations without parsing their contents. This document explains how File Sync works, which connectors support it, and how to use it.
+
+## Overview
+
+Traditional data integration in Airbyte involves extracting structured data as individual records, which are then processed and loaded into a destination. However, many use cases require transferring raw files without parsing their contents:
+
+- Moving binary files (images, videos, PDFs)
+- Transferring compressed files (ZIP, GZIP)
+- Migrating unstructured text data
+- Preserving file formats for specialized processing
+
+File Sync addresses these needs by copying files exactly as they appear in the source to the destination, preserving their original format and content.
+
+## How File Sync Works
+
+When using File Sync:
+
+1. The source connector identifies files to be transferred
+2. Instead of parsing file contents into records, the file is transferred as-is
+3. The destination connector writes the raw file to the target location
+4. File metadata (name, path, size, etc.) is preserved
```

Comment:
```suggestion
1. The source connector identifies files to be transferred.
2. Instead of parsing file contents into records, the file is transferred as-is.
3. The destination connector writes the raw file to the target location.
4. File metadata (name, path, size, etc.) is preserved.
```